thirdparty/vars.sh

```diff
@@ -415,9 +415,27 @@ CLUCENE_NAME="starrocks-clucene-2023.11.28.tar.gz"
 CLUCENE_SOURCE="starrocks-clucene-2023.11.28"
 CLUCENE_MD5SUM="d71f8156bbcbe78e5064ec5616072a5a"
 
+# libunistring
+LIBUNISTRING_DOWNLOAD="https://ftp.gnu.org/gnu/libunistring/libunistring-1.2.tar.gz"
```

Comment:
Are there other simpler code implementations that can replace these three dependent libraries? Introducing 3 dependencies for one function is a bit heavy